1209/EE00/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: SODA(SWD OpenSource Debug Adapter)
+owner: Explore Embedded
```

Comment:
This needs to be the same as the directory name for your org - in this case, 'ExploreEmbedded' without the space.